app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -212,6 +212,13 @@ def finish(*_args)
 
   alias_method :abort_job, :cleanup
 
+  def queue_callback(state, msg, result)
```

Comment:
- how about ```quque_abort_after_timeouts``` as a name?
- If the last state was abort and it had and keeps having timeouts we will create a loop here